dask_deltatable/core.py

```diff
@@ -319,72 +248,12 @@ def read_delta_table(
     return resultdf
 
 
-def read_delta_history(
-    path: str,
-    limit: int | None = None,
-    storage_options: dict[str, str] | None = None,
-    delta_storage_options: dict[str, str] | None = None,
-) -> dd.core.DataFrame:
-    """
-    Run the history command on the DeltaTable.
-    The operations are returned in reverse chronological order.
-
-    Parallely reads delta log json files using dask delayed and gathers the
-    list of commit_info (history)
-
-    Parameters
-    ----------
-    path: str
-        path of Delta table directory
-    limit: int, default None
-        the commit info limit to return, defaults to return all history
-
-    Returns
-    -------
-        list of the commit infos registered in the transaction log
-    """
-
-    dtw = DeltaTableWrapper(
-        path=path,
-        version=None,
-        columns=None,
-        storage_options=storage_options,
-        delta_storage_options=delta_storage_options,
+def read_delta_table(*args, **kwargs):
+    warnings.warn(
+        "`read_delta_table` was renamed, use `read_deltalake` instead",
+        category=DeprecationWarning,
```

Comment:
I think this is not necessary. Just remove this. The library hasn't been out for long enough for us to care about this. The last release was more than two years ago

Reply:
Ok.